spread.yaml

```diff
@@ -127,22 +127,6 @@ backends:
                 workers: 4
                 image: centos-8-64
 
-    google-tpm:
-        type: google
-        key: '$(HOST: echo "$SPREAD_GOOGLE_KEY")'
-        location: computeengine/us-east1-b
-        halt-timeout: 2h
-        systems:
-            - ubuntu-18.04-64:
-                workers: 1
-                image: ubuntu-1804-64-tpm-enabled
```

Comment:
So all available images support TPM now right?

Reply:
yes, xenial, bionic, eoan and focal support uefi. When the images are created by ubuntu-cloud-os project they are setting the UEFI_COMPATIBLE flag on gce now.

Reply:
Nice